src/personio_py/client.py

```diff
@@ -291,28 +289,94 @@ def get_attendances(self, employees: Union[int, List[int], Employee, List[Employ
         return self._get_employee_metadata(
             'company/attendances', Attendance, employees, start_date, end_date)
 
-    def create_attendances(self, attendances: List[Attendance]):
+    def create_attendances(self, attendances: List[Attendance]) -> bool:
         """
-        placeholder; not ready to be used
+        Create all given attendance records.
+
+        Note: Attendances are created sequentially. This function stops on first error.
+        All attendance records before the error will be created, all records after the error will be skipped.
```

Comment:
were you able to verify that this works? The [documentation](https://developer.personio.de/reference#post_company-attendances) does not indicate what happens when there are errors in only some of the attendance records.

Reply:
This is not documented but I verified this using a personio test account

Reply:
in this case, maybe we should tell the user which attendances have been created and which have failed? We could return a dictionary like

```
{
  'success': [list of attendance objects that were created],
  'failure': [list of attendance objects that were not created],
}
```

Reply:
I only did a very quick test of this behavior, but IIRC the response does not contain the information where the error happened.